### DISCUSSION TITLE
EPIDEMIOLOGY updates

### DISCUSSION THREAD

**Discussion by Goonigh2020** [neherlab/covid19_scenarios#526] [General]
Hi,
In previous version of model we could select region and speed which gave us diffent options.
So I can not find this option in new version? Does it mean that these inputs are the same for all regions?

**Comment by nnoll** [comment]
Hey @Goonigh2020 yes this option has been deprecated, but in favor of epidemiological estimates tailored for each region/scenario. So, for example, our given R0 parameters differs for New York vs California. Hope this helps!
